integration_tests/sdk/setup_integration.py

```diff
@@ -260,24 +260,53 @@ def setup_compute_integrations(client: Client, filter_to: Optional[str] = None)
     else:
         compute_integrations = list_compute_integrations()
 
-    # No need to do any setup for the demo db.
-    if "aqueduct_engine" in compute_integrations:
-        compute_integrations.remove("aqueduct_engine")
-
     if len(compute_integrations) == 0:
         return
 
     connected_integrations = client.list_integrations()
     for integration_name in compute_integrations:
         # Only connect to integrations that don't already exist.
         if integration_name not in connected_integrations.keys():
-            integration_config = _fetch_integration_credentials("compute", integration_name)
+            print(f"Connecting to {integration_name}")
+            if "aqueduct_engine" in compute_integrations:
+                # Connect to conda if specified, otherwise, do nothing for aq engine.
+                aq_config = _parse_config_file()["compute"][integration_name]
+                if aq_config and "conda" in aq_config:
+                    client.connect_integration(
+                        integration_name,
```

Comment:
wait so this means the conda integration will always be named "aqueduct_engine"? There might be a collision here once we add "aqueduct_engine" to the startup script the same way we do "aqueduct_demo". Can we use the user supplied name ala "conda: <conda integration name>"?

Reply:
I see..I tried to use `conda` but we have a few assumptions in our testing framework that makes things trickier to change. We have a few places that assumes `integration_name` are keys of the "compute" field, and we check for things like global engine config for non-aqueduct engines, which is not applicable for conda.

Reply:
Overall, there are a lot works entailed in the framework to use a different name here

Reply:
Wait I think there might be a misunderstanding. I'm trying to propose this:

```
compute:
    aqueduct_engine:
          conda: <conda integration name> # use this integration name instead
```

This is mostly just a config file parsing change right?

Reply:
sg! https://github.com/aqueducthq/aqueduct/pull/1216/commits/5372a2f19b8f41178a6813ff629f2caa5827d037 this should cover it